python/composio/tools/toolset.py

```diff
@@ -416,7 +416,7 @@ def add_auth(
             parameters=parameters,
         )
 
-    def check_connected_account(self, action: ActionType) -> None:
+    def check_connected_account(self, action: ActionType, entity_id: str) -> None:
```

Comment:
The docstring should be updated to include the new `entity_id` parameter and its purpose. Consider adding:
```python
"""
Args:
    action (ActionType): The action to check authorization for
    entity_id (str): The ID of the entity to check connected accounts for
"""
```